### DISCUSSION TITLE
QR코드 월드 테스트 접속 문제

### DISCUSSION THREAD

**Discussion by jangh4303** [naverz/zepeto-studio-kor#591] [5. Q&A]
안녕하세요 이제 처음 시작하게 된 뉴비입니다.
제페토 월드 패키지 : 1.1.5
제페토 앱 버전 : 3.11.0
개발 환경 : 노트북 window 10, 핸드폰 iOS 15.4.1
문제:
월드를 모바일에서 테스트 하기 위해서 QR코드를 읽었는데 핸드폰에서 파란 화면 이후 접속 오류라고 발생했습니다.
뭐가 문제인지 모르겠습니다 도와주세요 ㅠㅠ

![image](https://user-images.githubusercontent.com/94778069/161908879-4e25eaee-9407-48df-a63c-c3b257491aaa.png)

![image](https://user-images.githubusercontent.com/94778069/161908893-5f2a2d94-b82d-4a23-9c8b-f8c11477c0f5.png)

![image](https://user-images.githubusercontent.com/94778069/161909097-2b68feb6-4de8-4f3b-89fa-4ebd8f8165fd.png)

**Comment by pumfit** [comment]
같은 네트워크 환경인지 방화벽 설정이 풀어져있는지 먼저 확인해보시면 좋을 것 같아요!
아래는 가이드 링크입니다!
https://studio.zepeto.me/guides/getting-started-with-templates

_NOTE: QR코드를 통한 테스트는 스마트폰과 개발 PC가 같은 네트워크 환경에 접속되어야 동작됩니다.
Windows PC에서 연결이 원활하지 않을 경우 방화벽 설정을 변경하여 확인합니다._

**Reply by jangh4303** [reply]
답변해주셔서 정말 감사합니다ㅠㅠ 최대한 찾아봤는데 그래도 안되네요....

![image](https://user-images.githubusercontent.com/94778069/162555908-a7d3fcdf-145b-4c8d-b39d-4d1aa15dae87.png)
![image](https://user-images.githubusercontent.com/94778069/162555975-2b6cdd6d-0d99-404c-8827-edabc5ee6f55.png)
![image](https://user-images.githubusercontent.com/94778069/162556002-266328ad-4491-4d96-b5bb-d10fa080cd0c.png)
![image](https://user-images.githubusercontent.com/94778069/162556027-c0c81c6d-20c9-4994-a1c7-e28404d80b42.png)
![image](https://user-images.githubusercontent.com/94778069/162556079-3642c7df-d7a4-42f8-b5b6-5faf7cef118c.png)

제가 방화벽 설정 풀고 핸드폰과 노트북 동일 공유기로 네트워크 연결 했는데도 안되는데 혹시 제가 설정을 잘못 했나요..?

**Reply by pumfit** [reply]
안녕하세요 보내주신 답변을 보았는데 첫 사진에 나와있는 빌드된QR에 표시된 IP주소 와 진행하신 폰의 와이파이 네트워크의 IP주소가 달라 보이긴 합니다. 아래와 같이 빌드된 후 우측 하단에 IP주소가 표시되는데 이 주소와 와이파이 주소가 앞 세그룹만 동일한지 확인해주시면 될 것 같습니다. (답장 주신 정보들은 삭제해주시는 게 더 좋을 것 같습니다!)
![image](https://user-images.githubusercontent.com/46295539/162647893-4345f8cd-13e2-463a-be2f-043ad25472af.png)

**Reply by minjujuu** [reply]
#226 
추가로 사진과 같이 IP 주소 부분을 누르면 네트워크 목록이 뜨는데요~
모바일에 연결된 와이파이 확인해보시고 맞춰서 선택해주시면 될 것 같습니다!
해당 리스트에서 네트워크 변경한 후에도 잘 안된다면 Rebuild 해보시면 좋을 것 같습니다.
![image](https://user-images.githubusercontent.com/79794414/162862143-faa1ce43-b9be-49b3-a8da-37a6167957ea.png)

**Comment by 17beans** [comment]
안녕하세요, 방화벽 설정을 하셨지만 그래도 혹시 동작이 다를 수 있으니 제가 사용하는 방법 하나 알려드리겠습니다.
저도 윈도우를 사용하고 있는데요, 방화벽 설정은 별도로 하지 않고 네트워크를 공용 네트워크가 아닌 개인 네트워크로 설정하여 접속 문제를 해결하고 있습니다. 아래 이미지들을 참고해주시면 좋을 것 같습니다.

1. 우측 하단 알림 아이콘 영역에서 네트워크 아이콘 마우스 우클릭 -> 네트워크 및 인터넷 설정 열기
![image](https://user-images.githubusercontent.com/29908722/167285576-d05918a7-7a4c-4718-a647-b2dd848c8531.png)

2. 연결된 네트워크(여러 네트워크를 사용 중이시라면 로컬 서버 테스트에 사용할 네트워크)의 속성 버튼 클릭
![image](https://user-images.githubusercontent.com/29908722/167285620-a46ef8ff-d075-488a-9ad3-f56f36a6a128.png)

3. 공용이었다면 모든 네트워크에서 로컬 서버 테스트가 불가능했었습니다. 개인으로 변경해줍니다.
![image](https://user-images.githubusercontent.com/29908722/167285631-056b2d38-b300-4c33-beab-d148e0aa8143.png)

4. 변경 직후 10초 내외로 바로 접속이 되더군요. 만약 그래도 동작하지 않으시면 공유기 설정에서 문제가 있을 수 있을 것 같습니다. 참고로 윈도우 10의 핫스팟 기능을 통해서는 로컬 서버 테스트는 불가능합니다.
